app/javascript/components/TradeProposal/TradeProposalButtonSmall.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {useState} from 'react';
+import Button from '../Button';
+import TradeProposalModal from './TradeProposalModal';
+
+const TradeProposalButtonSmall = ({ card, currentUserId }) => {
+  const [modalOn, setModalOn] = useState(false);
+  return (
+    <>
+      <Button className={"trade-proposal-btn__small"} onClick={() => setModalOn(true)}>
```

Comment:
Quick BEM syntax note
```suggestion
      <Button className={"trade-proposal-btn--small"} onClick={() => setModalOn(true)}>
```